docs/source/reference/oauth.md

```diff
@@ -56,8 +56,8 @@ A single oauth flow generally goes like this:
 
 1. A **browser** makes an HTTP request to an oauth **client**.
 2. There are no credentials, so the client _redirects_ the browser to an "authorize" page on the oauth **provider** with some extra information:
-   - the oauth **client id** of the client itself
-   - the **redirect uri** to be redirected back to after completion
+   - the oauth **client id** of the client itself.
```

Comment:
fwiw, when bullets/lists are sentence _fragments_, adding a period on the end is not required or necessarily correct. But since _one_ item here is multiple sentences, I think this is okay here.

Reply:
I'll take note of that, thank you so much!